app/src/androidTest/java/org/sil/hearthis/MainActivityTest.java

```diff
@@ -1,125 +1,121 @@
 package org.sil.hearthis;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.CoreMatchers.*;
+// Test code basically all became obsolete with SDK 33. Don't know if it can be repaired.
```

Comment:
It would be nice. Android automated testing is really hard if you go anywhere near the UI, because they more-or-less force you to mock everything the OS normally does. I don't know when I will get more time to work on this, or whether testing will be a priority.